.travis.yml

```diff
@@ -6,30 +6,32 @@ services:
 dist: trusty
 
 go:
-  - "1.11.x"
-  - "1.10.x"
   - "tip"
 
 env:
   global:
     - IMAGINARY_VERSION="${TRAVIS_TAG:-dev}"
```

Comment:
By reading [this](https://docs.travis-ci.com/user/environment-variables/), `TRAVIS_BRANCH` sounds more useful. We should probably even consider commit sha if we want a incremental public build at some point

Reply:
This is mostly for `--build-arg`. It defaults on the value `dev` when no tag is used. 

The idea behind it is that Imaginary's `main.Version` is set to this value and I figured it matters only if we have a real release or not, in which case a generalised value would be enough. So in any other case we can default to `dev`, indication that it's a development version.

So basically, end users should never see "dev" in the `-v` output.

I can pick a branch, but I'm not sure what that would add in terms of information, since it only lives in CI and should never result to an image on Docker hub. Releases should only happen on Git Tags, which could be a x.x.x or even a x.x.x-rc or any other thing. As long as we can trace it back to a Git tag.